.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!-- These sections are meant as guidance for you. If something doesn't fit, you can just skip it. -->
+
+## Summary
+
+<!--- Provide a general summary description of your changes -->
+
+## Details
+
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+
+<!--- Please add tests for changes to the code, otherwise we probably won't merge it -->
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how -->
+<!--- your change affects other areas of the code, etc. -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue).
+- [ ] New feature (non-breaking change which adds functionality).
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] It is my own work, its copyright is implicitly assigned to the project and no substantial part of it has been copied from other sources (including [Stack Overflow](https://stackoverflow.com/)). In rare occasions this is acceptable, like in CMake modules where the original copyright information should be kept.
+- [ ] I'm using the same code standards as the existing code (indentation, spacing, variable naming, ...).
+- [ ] I've added tests for my code.
+- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
```

Comment:
These are two separate statements and should be separate check list items too IMO. E.g.:
- [ ] I have verified that my change requires no change to the documentation.
- [ ] My change either requires no documentation change or I've updated the documentation accordingly.

This splits the verification task (producing no committable artifacts) from the documentation task.